src/framework/theme/styles/themes/_mapping.scss

```diff
@@ -1303,31 +1303,137 @@ $eva-mapping: (
   tooltip-danger-text-color: text-control-color,
   tooltip-shadow: shadow,
 
+  option-list-max-height: 20rem,
+  option-list-shadow: shadow,
+  option-list-border-style: solid,
+  option-list-border-width: 0,
+  option-padding: 0.4375rem 1.125rem,
+  option-line-height: 1.5rem,
```

Comment:
Text/font relates properties should be prefixed with `text`, so name should be `option-text-line-height`. Also, please group line-height, font-*, color, etc. properties together

Reply:
Fixed